app/Resources/views/page.html.twig

```diff
@@ -178,6 +179,59 @@
     {% for javascript in javascripts %}
         <script async src="{{ asset(javascript) }}"></script>
     {% endfor %}
+
+    {% if hypothesis ?? false and is_granted('FEATURE_CAN_USE_HYPOTHESIS') %}
+        <script>
+            if (!!document.createElement('div').dataset) {
+                window.hypothesisConfig = function () {
+                    return {
+                        branding: {
+                            accentColor: '#0288D1',
+                            appBackgroundColor: 'white',
+                            ctaBackgroundColor: '#0288D1',
+                            ctaTextColor: 'white',
+                            selectionFontFamily: 'Georgia, Times, serif',
+                            annotationFontFamily: 'Georgia, Times, serif'
+                        },
+                        services: [{
+                            apiUrl: 'https://hypothes.is/api/',
+                            authority: '{{ hypothesis_authority }}',
+                            grantToken: {% if app.user %}'{{ hypothesis_token(app.user) }}'{% else %}null{% endif %},
```

Comment:
I don't know what's in the token, but I guess no `'` characters

Reply:
JWT, so `a-zA-Z0-9-_.`